### DISCUSSION TITLE
Experimentalist sampler template ready to be reviewed

### DISCUSSION THREAD

**Discussion by musslick** [AutoResearch/autora#367] [Show and Tell]
@younesStrittmatter @hollandjg Alright, I adapted the theorist template to an experimentalist sampler template. I was wondering...
1. if you could take a brief look, just to check it all looks ok? https://github.com/AutoResearch/autora-experimentalist-sampler-template
2. How can I turn this repo into a template? It's still listed as a regular repo.

**Comment by younesStrittmatter** [comment]
You can turn it into a template in the settings on the github page of the repository
